catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -108,6 +94,14 @@
   private static final Integer DEFAULT_MAX_PAGE_SIZE = 1000;
 
   public static final Integer MAX_PAGE_SIZE = determineAndRetrieveMaxPageSize();
+  // Inject properties
```

Comment:
❓ Did you make all these formatting changes by hand, or through the formatter plugin?

Reply:
By accident I believe. I think i accidentally checked rearrange code